### ISSUE TITLE
Video preview not showing up

### ISSUE THREAD

**Issue by jhk115** [twilio/video-quickstart-swift#8]
The video preview on load is not showing up. It is only when a conversation is started the preview shows. Any ideas how to fix this?

**Comment by jhk115** [comment]
https://www.twilio.com/docs/api/video/guide/conversations#render-camera-preview

**Comment by jmkr** [comment]
I'm having the same problem. 

Calling `self.camera?.startPreview()` seems to have no effect after setting up and attaching the video track to the local media view.

Any help or workarounds you guys know of? Thanks in advance!

**Comment by cummack** [comment]
Thanks for the report @jmkr and @jhk115 - we are going to look into this asap.

**Comment by jhk115** [comment]
@cummack thanks!

**Comment by jmkr** [comment]
thanks @cummack ! looking forward to hearing back!

**Comment by cummack** [comment]
Hi @jmkr, @jhk115. We've since decided to replace this quickstart app with something new from scratch. That work should be completed soon. In the mean time, there is a reference app that we just posted which includes the functionality you're describing, and is implemented in Swift:

https://github.com/twilio/video-sample-ios-capture-render

The ultimate goal of the mentioned app is to demonstrate more advanced capture and rendering behavior, but I think you'll find what you're looking for. And this quickstart app will also be updated shortly. Thanks for your patience on this one.
